experimental/traffic-portal/src/app/shared/interceptor/alerts.interceptor.ts

```diff
@@ -41,7 +41,8 @@ export class AlertInterceptor implements HttpInterceptor {
 		return next.handle(request).pipe(tap(
 			r => {
 				if (Object.prototype.hasOwnProperty.call(r, "body") &&
-				    Object.prototype.hasOwnProperty.call((r as {body: unknown}).body, "alerts")) {
+				Object.prototype.hasOwnProperty.call((r as {body: unknown}).body, "alerts") &&
+				(r as {body: {alerts: Array<unknown>}}).body?.alerts !== null) { //Ignore alerts with null value
```

Comment:
the only thing I'd change about this is that your `?.` safe-access operator on the `alerts` property of `r.body` is unnecessary; the previous check ensures that `body` is non-null and has an `alerts` property